src/words/Word.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Letter from './Letter';
+import { map } from 'lodash'
+
+function Word(props) {
```

Comment:
This isn't a problem at all but just to a FYI: you are able to destructure directly from the function's `props` params

```suggestion
   function Word({ letters }) {
```

Reply:
thanks